utils/scripts/update_generated_code_from_idl.sh

```diff
@@ -3,9 +3,19 @@
 files_to_exclude=(
     './include/fastrtps/types/*'
     './include/fastdds/statistics/types.idl'
```

Comment:
If we are adding the other Fast DDS-Gen options to the IDL files with exceptions I suggest adding the final one as well. In this case the option to be set is `-d` followed by the output directory `${SOURCE_DIR}/src/cpp/statistics/types`

Reply:
Done in 67797b9